doc/nrf/ug_matter_overview_architecture.rst

```diff
@@ -88,5 +88,6 @@ Transport and IP Framing
 ************************
 
 This layer manages the transmission of the payload through the IP network to the peer device.
-It uses either the Transmission Control Protocol (TCP) or the combination of User Datagram Protocol (UDP) with Matter's Message Reliable Protocol (MRP).
-During the commissioning process, Bluetooth Transport Protocol over Bluetooth LE can be used instead of that layer.
+It uses either the Transmission Control Protocol (TCP) or the combination of User Datagram Protocol (UDP) with Matter's Message Reliability Protocol (MRP).
+The MRP ensures that duplicated messages are rejected.
```

Comment:
I guess it's not only that, right (i.e. MRP implements e.g. retransmissions)?

Reply:
Rephrased to "The MRP implements retransmissions, provides confirmation of delivery, and ensures that duplicated messages are rejected."